### PR TITLE
Debug performance test

### PR DESCRIPTION
**Pull request by alexey-milovidov**
Changelog category (leave one):
- Not for changelog (changelog entry is not required)

There are misterious timeouts in "prewarm" runs.

### CODE REVIEW

**Comment by alexey-milovidov** [comment]
We need some Python experts to review this PR.

**Comment by alexey-milovidov** [comment]
Maybe there is an issue with different Python version?

File: docker/test/performance-comparison/perf.py [hunk 1]
```diff
@@ -268,26 +269,41 @@ def do_create(connection, index, queries):
...
                 # * detect queries that are exceedingly long, to fail fast,
                 # * collect profiler traces, which might be helpful for analyzing
                 #   test coverage. We disable profiler for normal runs because
                 #   it makes the results unstable.
-                res = c.execute(q, query_id = prewarm_id,
-                    settings = {'max_execution_time': args.max_query_seconds,
-                        'query_profiler_real_time_period_ns': 10000000})
-            except clickhouse_driver.errors.Error as e:
+
+                proc = subprocess.run([
```
Comment (azat) [inline docker/test/performance-comparison/perf.py:284]: Why it uses clickhouse-client over clickhouse_driver (python) ?
Does this hunk fixes something?

**Comment by alexey-milovidov** [comment]
@azat We have misterious rare query hangs in performance test*.
This PR is needed to eliminate one possible cause - Python ClickHouse driver.

\* most likely it is described and fixed here: #25968

**Comment by alexey-milovidov** [comment]
Unfortunately, the parameters of `subprocess.run` method in Python are too much dependent on Python version.

**Comment by alexey-milovidov** [comment]
It looks like #25968 was the source of trouble and now it is fixed.
This PR is no longer relevant.

**Pull request closed without merging**